btr-api/migrations/versions/2049364ea0a6_.py

```diff
@@ -0,0 +1,71 @@
+"""empty message
```

Comment:
Hey patrick can you fix your branch so that it doesn't include hrvoje's stuff as a new commit ?

Reply:
if you need help rebasing let me know